infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -283,9 +283,13 @@ private void handleException(Throwable ex, ChannelHandlerContext ctx, RequestMsg
     @RequestHandler(type = PayloadCase.WRITE_LOG_REQUEST)
     private void handleWrite(RequestMsg req, ChannelHandlerContext ctx, IServerRouter router) {
         LogData logData = getLogData(req.getPayload().getWriteLogRequest().getLogData());
-
         log.debug("handleWrite: type: {}, address: {}, streams: {}",
-                logData.getType(), logData.getToken(), logData.getBackpointerMap());
+                logData.getType(), logData.getToken(),
+                logData.getBackpointerMap().entrySet().stream().collect(Collectors.toMap(k ->
+                                serverContext.getStreamNames().getOrDefault(k, k.toString()),
+                        Map.Entry::getValue) // Re-map to stream name -> long from UUID -> long
```

Comment:
We should keep at least a portion of the UUID since that's what's used throughout the system (i.e., sequencer logs, LogUnit startup logs, etc...).

Reply:
Why not the other way around - use the table name throughout the system. Humans don't understand UUID and if we are logging UUIDs that's already painful debugging by design.